src/main/java/Duke.java

```diff
@@ -1,10 +1,36 @@
+import java.io.FileNotFoundException;
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    private Storage storage;
+    private Parser parser;
+    private UI ui;
+    private TaskList taskList;
+
+    public Duke(String filepath) throws Exception {
```

Comment:
Will it be a better idea to throws a specific exception instead of a general Exception?

Reply:
Dear Jun Hong T, 

I agree that throwing a specific exception would be however, I believe that it is imperative for me to cover every base should there be any exception that passed my notice. Though this could just be unnecessary paranoia. Despite that, thank you for point this out to me! My sincerest gratitude for your review Jun Hong T, very epic!

Best regards
Roby Tanama